### PR TITLE
[css-view-transitions-2] Allow pseudo-element with class and no *

### LINKED ISSUE #9874: [css-view-transitions-2] Consider allowing `::view-transition-group(.class)` without *
This would be consistent with how class selectors work outside of view-transitions.

### PR DESCRIPTION
**Pull request by noamr**
Based on resolution:
https://github.com/w3c/csswg-drafts/issues/9874#issuecomment-2023235242

Closes #9874

### CODE REVIEW

File: css-view-transitions-2/Overview.bs [hunk 1]
```diff
@@ -631,17 +631,18 @@ div.box {
...
 	</pre>
 
 	where <<pt-name-selector>> works as previously defined, and
-	<<pt-class-selector>> has the following syntax definition:
+	<<pt-name-and-class-selector>> has the following syntax definition:
 
 	<pre class=prod>
-		<dfn>&lt;pt-class-selector></dfn> = ['.' <<custom-ident>>]*
+		<dfn>&lt;pt-name-and-class-selector></dfn> = <<pt-name-selector>><<pt-class-selector>>? | <<pt-class-selector>>
+		<dfn>&lt;pt-class-selector></dfn> = ['.' <<custom-ident>>]+
```
Comment (khushalsagar) [inline css-view-transitions-2/Overview.bs:645]: Confirming, the "+" means there needs to be at least 1 class.

File: css-view-transitions-2/Overview.bs [hunk 2]
```diff
@@ -631,17 +631,18 @@ div.box {
...
+		::view-transition-new(<<pt-name-and-class-selector>>)
 	</pre>
 
 	where <<pt-name-selector>> works as previously defined, and
-	<<pt-class-selector>> has the following syntax definition:
+	<<pt-name-and-class-selector>> has the following syntax definition:
 
 	<pre class=prod>
-		<dfn>&lt;pt-class-selector></dfn> = ['.' <<custom-ident>>]*
+		<dfn>&lt;pt-name-and-class-selector></dfn> = <<pt-name-selector>><<pt-class-selector>>? | <<pt-class-selector>>
```
Comment (khushalsagar) [inline css-view-transitions-2/Overview.bs:644]: And here "?" implies zero or more class selectors.

**Review by khushalsagar — APPROVED** [review]

File: css-view-transitions-2/Overview.bs (hunk 1 above)
Comment (noamr) [reply css-view-transitions-2/Overview.bs:645]: Yes

File: css-view-transitions-2/Overview.bs (hunk 2 above)
Comment (noamr) [reply css-view-transitions-2/Overview.bs:644]: Yes, zero or one `<<pt-class-selector>>`, which would be one or more (because it has +)

**Pull request merged**